llvm/docs/ReleaseNotes.rst

```diff
@@ -235,11 +235,25 @@ AIX improvements
 Changes to the RISC-V Backend
 -----------------------------
 
+* :doc:`RISCVUsage` was introduced to document the status of support within
```

Comment:
Interesting, I don't think this was part of the original fix. Is this an artifact of creating a merge PR automatically?

Reply:
Yes it, must be. In the original commit, there is nothing like this.

Reply:
We should drop the change for this file.

Reply:
You can ignore this. It's an artifact of our automatic PR system and it will go away when we sync and merge.